ui/src/clientLibraries/components/ClientGoOverlay.tsx

```diff
@@ -6,10 +6,40 @@ import ClientLibraryOverlay from 'src/clientLibraries/components/ClientLibraryOv
 
 // Constants
 import {clientGoLibrary} from 'src/clientLibraries/constants'
+import CodeSnippet from 'src/shared/components/CodeSnippet'
 
 const ClientGoOverlay: FunctionComponent<{}> = () => {
   const {name, url} = clientGoLibrary
+  const initializeClientCodeSnippet = `
```

Comment:
These should be hoisted out of the render method (to line 10ish) for a small perf upgrade as they don't reference props

Reply:
Also why aren't they defined in the constants file in the same way as in the C# example?

Reply:
WIP